com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/StepsUtil.java

```diff
@@ -985,23 +985,11 @@ static CloudModelConfiguration getCloudBuilderConfiguration(DelegateExecution co
     }
 
     static String getGitRepoRef(DelegateExecution context) {
-        Object gitRepoConfigObject = context.getVariable(Constants.VAR_GIT_REPOSITORY_CONFIG_MAP);

```

Comment:
I like this, but was the deleted code really unneeded?

Reply:
Looool something happened - in my codebase there is no such thing o.O